test/e2e/standard/standard_test.go

```diff
@@ -1,53 +1,95 @@
 package standard
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	"io/ioutil"
+	"path/filepath"
 	"time"
 
+	kvrocksv1alpha1 "github.com/RocksLabs/kvrocks-operator/api/v1alpha1"
+	"github.com/RocksLabs/kvrocks-operator/pkg/client/kvrocks"
+	"github.com/RocksLabs/kvrocks-operator/pkg/resources"
+	. "github.com/RocksLabs/kvrocks-operator/test/e2e/util"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	kruise "github.com/openkruise/kruise-api/apps/v1beta1"
 	corev1 "k8s.io/api/core/v1"
+
 	k8serr "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	k8syaml "k8s.io/apimachinery/pkg/runtime/serializer/yaml"
 	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
+	"k8s.io/client-go/util/homedir"
+	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/yaml"
+)
 
-	kvrocksv1alpha1 "github.com/RocksLabs/kvrocks-operator/api/v1alpha1"
-	"github.com/RocksLabs/kvrocks-operator/pkg/client/kvrocks"
-	"github.com/RocksLabs/kvrocks-operator/pkg/resources"
-	"github.com/RocksLabs/kvrocks-operator/test/e2e/suite"
+var (
+	env           *KubernetesEnv
+	ctx           context.Context
+	kvrocksClient *kvrocks.Client
 )
 
-var _ = Describe("KVRocks standard controller", func() {
+// TODO remove the hard code and use the config.yaml
```

Comment:
I believe that currently 'Cluster setup' and 'Only code' are essentially the same, both creating a cluster through minikube for testing. If we wanted to provide two options, they should be creating a cluster and using an existing cluster. Of course, the config.yaml is in a 'to-do' state.

Reply:
@ColinChamber Hi,
Now we can use the `config.yaml` file.